reference/user_defined_functions.rst

```diff
@@ -355,6 +355,33 @@ Example
           7 |       7 | new-node |    12345 | default  | f           | t        |       0 | primary  | default
     (1 row)
 
+
+master_update_node
+$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$$
+
+The master_update_node() function changes the hostname and port for a node registered in the Citus metadata table :ref:`pg_dist_node <pg_dist_node>`.
```

Comment:
This came up in a few customer conversations. Could you also add an FAQ Entry for this? What scenarios would one want to do this?

Reply:
✅